internal/pkg/saelastic/client.go

```diff
@@ -60,7 +60,7 @@ func (ec *ElasticClient) InitAllMappings() {
 
 // createHttpsClient creates http.Client for elastic.Client with enabled
```

Comment:
I've initially named this createHttpsClient and then renamed. Maybe we can fix the comment in this patch.

Reply:
I'll fix this now, thanks for noting it!